packages/eslint-plugin-pf-codemods/test/rules/v5/selectDropdown-rename-itemId.js

```diff
@@ -0,0 +1,112 @@
+const ruleTester = require("../../ruletester");
+const rule = require("../../../lib/rules/v5/selectDropdown-rename-itemId");
+
+ruleTester.run("selectDropdown-rename-itemId", rule, {
+  valid: [
+    {
+      code: `import { SelectOption } from '@patternfly/react-core'; <SelectOption someProp />`,
+    },
+    {
```

Comment:
not a big deal but /next for valid tests too? not sure that `someProp` test has much value alongside `value`